src/python/CRABClient/Commands/setfilestatus.py

```diff
@@ -0,0 +1,150 @@
+# pylint: disable=consider-using-f-string, unspecified-encoding
+"""
+allow users to (in)validate some files in their USER datasets in phys03
+"""
+
+import json
+
+from CRABClient.Commands.SubCommand import SubCommand
+from CRABClient.ClientExceptions import MissingOptionException, ConfigurationException, CommandFailedException
+from CRABClient.ClientUtilities import colors
+from CRABClient.RestInterfaces import getDbsREST
+
+
+class setfilestatus(SubCommand):
+    """
+    Set status of a USER dataset in phys03,
```

Comment:
this command does not touch dataset status, right?

Reply:
right